index.js

```diff
@@ -28,7 +28,8 @@ function validateCustomConfig(config) {
 export default function ({
 	debug = false,
 	customStaticWebAppConfig = {},
-	esbuildOptions = {}
+	esbuildOptions = {},
+	allowedRoles = ['anonymous']
```

Comment:
I think the default should be undefined:
```suggestion
	allowedRoles
```